game.py

```diff
@@ -1,80 +1,50 @@
-from console import *
-from board import *
-from score import *
-from player import *
-from turn import *
-from validation import *
+class Game:
```

Comment:
Nice job decoupling this class by instantiating your other classes outside of Game! What do you think about passing in those objects as parameters when you create a new instance of the Game class? This way we'd have access to them as attributes of the Game class, and could avoid having to pass them each time we call a method

Reply:
that was how I had them being passed in when I originally decoupled it. But I remember someone suggesting I pass them in through play_game because of the difficulty of testing it when they are passed through when the class is initialized? I'm down to change it back. It would save a ton of headaches on my part as far as creating all those methods lol